src/transformers/models/marian/__init__.py

```diff
@@ -42,7 +42,7 @@
     ]
 
 if is_tf_available():
-    _import_structure["modeling_tf_marian"] = ["TFMarianMTModel"]
+    _import_structure["modeling_tf_marian"] = ["TFMarianMTModel", "TFMarianModel"]
```

Comment:
Should be imported in the `if TYPE_CHECKING` part too.

Reply:
great catch!